artview/modes_legacy.py

```diff
@@ -217,6 +220,21 @@
     )
 
 
+cmap_radar_mode = (
+        [FileNavigator, RadarDisplay, ColormapEdit],
+        [
+            ((1, 'Vcolormap'), (2, 'Vcolormap')),
+            ]
+        )
+
+
+cmap_grid_mode = (
+        [FileNavigator, GridDisplay, ColormapEdit],
+        [
+            ((1, 'Vcolormap'), (2, 'Vcolormap')),
+            ]
+        )
+
```

Comment:
Not the placement, but I guess this is the same as "if you want to use legacy interface..."